android/app/src/main/java/com/mattermost/helpers/PushNotificationDataHelper.kt

```diff
@@ -33,83 +36,73 @@ class PushNotificationDataRunnable {
             mutex.withLock {
-                val serverUrl: String = initialData.getString("server_url") ?: return
+                val serverUrl: String = initialData.getString("server_url") ?: return null
                 val db = dbHelper.getDatabaseForServer(context, serverUrl)
+                var result: Bundle? = null
 
                 try {
-                    val teamId = initialData.getString("team_id")
-                    val channelId = initialData.getString("channel_id")
-                    val rootId = initialData.getString("root_id")
-                    val isCRTEnabled = initialData.getString("is_crt_enabled") == "true"
-                    Log.i("ReactNative", "Start fetching notification data in server=$serverUrl for channel=$channelId")
-
                     if (db != null) {
-                        var teamData: ReadableMap? = null
-                        var myTeamData: ReadableMap? = null
-                        var channelData: ReadableMap? = null
-                        var myChannelData: ReadableMap? = null
-                        var loadedProfiles: ReadableArray? = null
-                        var postData: ReadableMap?
-                        var posts: ReadableMap? = null
-                        var userIdsToLoad: ReadableArray? = null
-                        var usernamesToLoad: ReadableArray? = null
-
-                        var threads: ReadableArray? = null
-                        var usersFromThreads: ReadableArray? = null
-                        val receivingThreads = isCRTEnabled && !rootId.isNullOrEmpty()
+                        val teamId = initialData.getString("team_id")
+                        val channelId = initialData.getString("channel_id")
+                        val postId = initialData.getString("post_id")
+                        val rootId = initialData.getString("root_id")
+                        val isCRTEnabled = initialData.getString("is_crt_enabled") == "true"
 
-                        coroutineScope {
-                            if (teamId != null && !TextUtils.isEmpty(teamId)) {
-                                val res = fetchTeamIfNeeded(db, serverUrl, teamId)
-                                teamData = res.first
-                                myTeamData = res.second
-                            }
+                        Log.i("ReactNative", "Start fetching notification data in server=$serverUrl for channel=$channelId")
 
-                            if (channelId != null) {
-                                val channelRes = fetchMyChannel(db, serverUrl, channelId, isCRTEnabled)
-                                channelData = channelRes.first
-                                myChannelData = channelRes.second
-                                loadedProfiles = channelRes.third
+                        val receivingThreads = isCRTEnabled && !rootId.isNullOrEmpty()
+                        val writableMap = Arguments.createMap()
```

Comment:
Better naming?

Reply:
what is wrong with the name? and in fact that was the name that was there before.

Reply:
Maybe I am missing the point of the variable, but `writableMap` seems to be referring to the type and not the functionality of the variable. Like doing `const set = new Set()` instead of `const channelsSet = new Set()`.

Here the naming may be something like `bundleMap` or something similar? Something that has more to do with the functionality of the variable.

Reply:
I see what you mean,.. ok I'll change it to something "better"

Reply:
done.